### ISSUE TITLE
How to use htmlunit-rhino-fork insteadof htmlunit-core-js?

### ISSUE THREAD

**Issue by inshua** [HtmlUnit/htmlunit-core-js#3]
I want try htmlunit-rhino-fork, please help

**Comment by rbri** [comment]
Sorry inshua,

there is a misunderstanding  regarding this repositories. Htmlunit-rhino-fork is a fork of the latest rhino stuff and only there to maintain small enhancements required only for HtmlUnit. Htmlunit-core-js uses this code and only changes the packages to avoid conflicts.
In fact if you use the latest core-js you have the latest version from this repo.
And i try to have all this in sync.

To sum it up - if you like to have the latest rhino stuff use the latest core-js.

If you think you have a problem please open an issue for HtmlUnit and i will try to have a look / find a fix.

Thanks
